### ISSUE TITLE
[kernel] /proc/kallsyms still needed by default?

### ISSUE THREAD

**Issue by BryanQuigley** [sosreport/sos#2482]
Was looking at the biggest files we always collect and kallsyms stood out a lot.  Is this useful enough to be in default collection?  (My understanding - which could be wrong - is that a lot of this could be looked up for that kernel build via system.map).

It's 33% of the compressed total of a basic sosreport (4.35MiB -> 2.93MiB).

**Comment by pmoravec** [comment]
I got some feedback from our kernel support colleagues and they would like to keep the collection (at least for RedHat distros), as they sometimes see the info useful.

**Comment by TurboTurtle** [comment]
Yep, got the roughly the same feedback - desired as a default collection.

So if it's really undesired as a default for Ubuntu runs, we can split out the kernel plugin and have a `RedHatKernel` that keeps it default and a `UbuntuKernel` that adds a plugin option (or just drop it and have on the RH override, your call).

**Comment by BryanQuigley** [comment]
Thanks! Just thought it was worth checking as it's such a large percentage - and it was added a while ago.
